lib/rng-browser.js

```diff
@@ -3,29 +3,34 @@
 // and inconsistent support for the `crypto` API.  We do the best we can via
 // feature-detection
 
-var getRandomValues = (typeof(crypto) != 'undefined' && crypto.getRandomValues) ||
-                      (typeof(msCrypto) != 'undefined' && msCrypto.getRandomValues);
-if (getRandomValues) {
-  // WHATWG crypto RNG - http://wiki.whatwg.org/wiki/Crypto
-  var rnds8 = new Uint8Array(16); // eslint-disable-line no-undef
+var rng = /*#__PURE__*/function () {
+  var getRandomValues = typeof crypto != 'undefined' && crypto.getRandomValues || typeof msCrypto != 'undefined' && msCrypto.getRandomValues;
```

Comment:
This is missing binding: https://github.com/kelektiv/node-uuid/pull/257

Reply:
well, it wasnt there when ive created/rebased this PR :P need to rebase again, gonna do that in a moment